toolchest_client/tools/api.py

```diff
@@ -805,7 +805,7 @@ def lastal5(output_path=None, output_primary_name="out.maf", inputs=[], database
 
 def lug(script, tool_version, custom_docker_image_id, container_name, docker_shell_location, inputs=None,
         output_path=None, tool_args="", instance_type=InstanceType.COMPUTE_2, volume_size=8, streaming_enabled=True,
-        pip_dependencies=None, **kwargs):
+        pip_dependencies=None, universal_name=None, **kwargs):
```

Comment:
Why is universal name exposed here, but not universal volume name? Is either needed, or can they be passed as kwargs?